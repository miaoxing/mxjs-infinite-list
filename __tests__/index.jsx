import {render} from '@testing-library/react';
import InfiniteList from '..';

describe('InfiniteList', () => {
  test('basic', () => {
    const result = render(<InfiniteList render={() => 'test'}/>);
    expect(result.container).toMatchSnapshot();
  });
});
