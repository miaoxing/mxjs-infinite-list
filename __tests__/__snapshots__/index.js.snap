// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfiniteList basic 1`] = `
<div>
  <div>
    test
  </div>
</div>
`;
